arduino-ide-extension/package.json

```diff
@@ -93,6 +93,7 @@
     "which": "^1.3.1"
   },
   "devDependencies": {
+    "@axe-core/react": "^4.3.2",
```

Comment:
Shouldn't it go under `dependencies` instead of `devDependencies`?

Reply:
Makes sense. Thanks!

Reply:
Thanks for the comment @kittaakos. Basically we're doing this to help the designers testing the app. We'll definitely change it as dev dependency as soon as we find a better solution.